.expeditor/build.docker.yml

```diff
@@ -0,0 +1,3 @@
+image_registry: chef
+env:
+  VERSION: "16.3.18"
```

Comment:
is this something we can just source from the VERSION file so we can maintain that as the source of truth for buildkite/expeditor actions

Reply:
I added that to the update-version.sh file already so it should be handled that way